.env.example

```
@@ -17,4 +17,7 @@ NEXT_PUBLIC_LOCAL_AUTH_COOKIE=true
 
 DESCI_RESEARCH_CONNECTION_DATABASE_URL=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)